src/avx/avx32_butterflies.rs

```diff
@@ -5,8 +5,8 @@ use std::mem::MaybeUninit;
 use num_complex::Complex;
 
 use crate::array_utils;
-use crate::array_utils::workaround_transmute_mut;
 use crate::array_utils::DoubleBuf;
+use crate::array_utils::*;
```

Comment:
I'm personally not a fan of * imports, because it obfuscates symbols' origins. Please manually list out the symbols being imported here.